test/api_integration/apis/general/csp.js

```diff
@@ -27,7 +27,7 @@ export default function ({ getService }) {
 
       const entries = Array.from(parsed.entries());
       expect(entries).to.eql([
-        ['script-src', ["'unsafe-eval'", "'self'"]],
+        ['script-src', ["'self'"]],
```

Comment:
I think this smoke test is actually redundant because we already make assertions about the CSP header value in the rendering service jest integration tests (`src/core/server/http_resources/integration_tests/http_resources_service.test.ts`)

I think we can just remove this test suite entirely, WDYT?

Reply:
Since this is an integration test it tests more of the request response life cycle as opposed to `src/core/server/http_resources/integration_tests/http_resources_service.test.ts` which is just a unit test. For that reason I think it's nice to have the better coverage that the integration test offers. Or am I putting too much value into it?

Reply:
Fixed in https://github.com/elastic/kibana/pull/124484/commits/ce3a317158ad17315fbf543f9eb3e463eebec332